Monika After Story/game/script-ch30.rpy

```diff
@@ -1526,16 +1526,16 @@ label ch30_post_mid_loop_eval:
 
         # Thunder / lightning if enabled
         if (
-                store.mas_globals.show_lightning
-                and renpy.random.randint(1, store.mas_globals.lightning_chance) == 1
-            ):
+            store.mas_globals.show_lightning
+            and renpy.random.randint(1, store.mas_globals.lightning_chance) == 1
+        ):
             $ light_zorder = MAS_BACKGROUND_Z - 1
             if (
-                    store.mas_globals.show_s_light
-                    and renpy.random.randint(
-                        1, store.mas_globals.lightning_s_chance
-                    ) == 1
-                ):
+                store.mas_globals.show_sayori_lightning
+                and persistent._mas_pm_cares_about_dokis
```

Comment:
Is there supposed to be a `not` on this doki care check?

Reply:
Yes you're right. Good catch